Source/RomanNumeral.c

```diff
@@ -6,11 +6,22 @@
 #include <stdio.h>
 #include "RomanNumeral.h"
 
-static char * Append(char *output, const char *toAppend)
+void Append(char *output, const char *toAppend)
 {
-   return output + sprintf(output, "%s", toAppend);
```

Comment:
This is actually a much faster way to do it for large strings. The reason is that you don't have to continuously search for the null terminator to figure out where you should insert.